docs/output.md

```diff
@@ -663,7 +680,7 @@ Plot will show:
 
 For all samples:
 
-**Output directory: `results/Reports/[SAMPLE]/BCFToolsStats`**
+**Output directory: `{outdir}/Reports/[SAMPLE]/BCFToolsStats`**
 
 - `VariantCaller_[SAMPLE].bcf.tools.stats.out`
```

Comment:
```suggestion
- `[sample].variantcaller.bcftools_stats.txt`
```

Reply:
Hey @asp8200 and @SusiJo! I stopped yesterday at VEP so no news for the QC tools yet. Everything there is still old and will be updated today